### PR TITLE
Added Desktop version portfolio feature

### PR DESCRIPTION
**Pull request by rebel216**
Added desktop version for portfolio app.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -144,17 +238,44 @@ button:hover {
   transition: box-shadow 0.4s ease-out;
 }
 
+.desk-work-list {
+  list-style: none;
+}
+
+@media (min-width: 768px) {
+  .desk-work-list {
```
Comment (danielufeli) [inline style.css:275]: - [x] Nice Working on the recent works section, kindly remove the `br` beak on the `My Recent Works` title so it looks like what is on the Figma Template.

- [ ] Great Working on the skills tags, kindly add padding space to the tags, so it aligns with the Figma Template.

## Figma Template
Yours

## Your Design
rworks-urs

File: style.css [hunk 2]
```diff
@@ -290,19 +394,68 @@ button:hover {
...
   margin: 1em auto;
-  height: 400px;
+  height: 667px;
   width: 327px;
   border-radius: 8px;
   list-style: none;
 }
 
+@media (min-width: 768px) {
+  .form {
```
Comment (danielufeli) [inline style.css:501]: - [ ] Nice work on your About section, kindly reduce the `white spacing` and add `padding` space to the tags so it aligns with what you have in the Figma Template.

## Figma Template
ab-space-fig

## Your Design
ab-space

File: style.css [hunk 3]
```diff
@@ -290,19 +394,68 @@ button:hover {
...
+    justify-content: center;
+    text-align: center;
+    margin: -30px;
+    width: 90%;
+    grid-auto-flow: column;
+    list-style: none;
+  }
+}
+
+.deskimg {
```
Comment (danielufeli) [inline style.css:575]: - [ ] Great work on your Footer section, kindly add the background image, a horizontal line above the social icon, padding to the social icons, and align the button to the left so it looks neater just like it is on the Figma Template.

- [ ] Kindly check the Mobile View of the footer the social icons and the e-mail appears twice.

## Your Design
f-bg

## Figma Template
f-bg-fig

## Mobile View
mview

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @rebel216,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 Alt text

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -26,284 +24,358 @@
...
-        </article>
-
-        <div class="headline">
-          <article class="work-card">
-            <img src="./asset/cat2.jpeg" alt="Tonic work" class="work-image" />
+    </section>
 
+    <div class="aboutmeimage">
+      <div class="form">
+        <h2>About Me</h2>
```
Comment (koomekevo) [inline index.html:177]: @rebel216 Kindly implement the padding around the about me section as requested by the previous reviewer. The below screenshot is the correct design 

rebelresumecorrect

The below screenshot is your design 

rebelresumewrongedited

File: index.html [hunk 5]
```diff
@@ -26,284 +24,358 @@
...
+              <button type="submit" class="btn-pop">Get In Touch</button>
             </li>
           </ul>
-
         </form>
 
-        <div class="container-flex">
-
-          <div class="social-media">
-
```
Comment (koomekevo) [inline index.html:278]: @rebel216 Nice job on adding the images as requested by the previous reviewer. However, you need to add the correct spacing between the get in touch button and one of the background images. Also please space the social media icons accordingly as per the Figma template. The screenshot below shows the correct design. 

rebelgetintouchiconscorrect

The below screenshot shows your design 

rebelgetintouchiconswrong

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @rebel216, 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koomekevo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: 
Hi @rebel216, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**